recipes/ftxui/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,27 @@
+#include <ftxui/dom/elements.hpp>  // for operator|, text, Element, hcenter, Fit, hbox, size, window, Elements, bold, dim, EQUAL, WIDTH
+#include <ftxui/screen/screen.hpp>  // for Screen
+#include <memory>                   // for allocator, shared_ptr
+#include <string>                   // for string, to_string
+#include <utility>                  // for move
+
+#include "ftxui/dom/node.hpp"      // for Render
+#include "ftxui/screen/color.hpp"  // for ftxui
```

Comment:
```suggestion
#include <ftxui/dom/elements.hpp>  // for operator|, text, Element, hcenter, Fit, hbox, size, window, Elements, bold, dim, EQUAL, WIDTH
#include <ftxui/screen/screen.hpp>  // for Screen
#include <memory>                   // for allocator, shared_ptr
#include <string>                   // for string, to_string
#include <utility>                  // for move
```
I think node.hpp and color.hpp are not necessary.
- node.hpp is included by elements.hpp
- color.hpp is included by screen.hpp
or 
Please include ftxui headers at one place and same format( "" or <> ).

Reply:
I actually took it from an example, except interactive parts which I could not include.